scripts/environment_json.sh

```diff
@@ -41,6 +42,8 @@ while getopts rp:h? opt; do
 done
 shift $((OPTIND-1))
 
+SEMVER=$1
```

Comment:
My intent was to rewire it so that the other platforms require it as well. Would you prefer I rolled this back and tackled that horizontally in a separate PR?